src/commands/clear.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const interfacer = require('./../util/interfacer');
+
+const clear = {
+  exec() {
+    this.log('\u001b[2J\u001b[0;0H');
+    return 0;
+  }
+};
+
+module.exports = function (vorpal) {
+  if (vorpal === undefined) {
+    return clear;
+  }
+  vorpal.api.clear = clear;
+  vorpal
+    .command('clear [files...]')
+    .action(function (args, callback) {
+      args.options = args.options || {};
+      return interfacer.call(this, {
+        command: clear,
+        args: args.files, // only pass in what you need from Vorpal
```

Comment:
Since there are no args, just change this to `args: args`. 

You can also remove those two comment lines.